### ISSUE TITLE
Find a database with serving sizes, do a look-up and append the serving size in parentheses next to the word "Serving"

### ISSUE THREAD

**Issue by AMelhede** [curedao/curedao-monorepo#39]
How can one know what a serving, count is in ml, grams, etc.?

For instance in these examples:
![image](https://user-images.githubusercontent.com/95473109/226525735-e175d719-97f2-4153-a4c2-3eb8c0ff4f84.png)

![image](https://user-images.githubusercontent.com/95473109/226525601-5f11d7a7-b076-4b9c-869a-a20e72a97389.png)

**Comment by mikepsinn** [comment]
It should say the serving size on the package. 

Otherwise, the easiest thing would be for people to just select the mL or grams unit if they don't know how much a serving is. 

If someone can find a database with the serving size for everything, they could do a look-up and append the serving size in parentheses next to the word "Serving".

**Comment by AMelhede** [comment]
Yes. The last paragraph you wrote is what I was thinking. That would be very helpful.
Especially, when you have to choose a category, but are not able to at this moment like with nutrients, supplements, and beverages

**Comment by mikepsinn** [comment]
Supplements currently go in the Treatments category and Beverages go in the Foods category. We can change the display names for those categories to make it more intuitive in #38 

The Nutrients category isn't displayed in the manual measurement recording because those are automatically calculated from the Foods recorded. So you shouldn't normally have to manually enter all the nutrients in your foods.

**Comment by AMelhede** [comment]
Okay, that sounds good.

What about things you add to your food like spices, herbs, salt, etc.?
